src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -2617,15 +2618,18 @@ namespace cryptonote
             KV_SERIALIZE(staking_requirement)
             KV_SERIALIZE(portions_for_operator)
             KV_SERIALIZE(operator_address)
+            KV_SERIALIZE(as_json)
         END_KV_SERIALIZE_MAP()
       };
 
       std::vector<entry> service_node_states;
       std::string        status;
+      std::string        sns_as_json;
```

Comment:
Descriptive variable names would be nice.

From earlier comment, we can remove the ```as_json``` part from entry, and instead name ```sns_as_json```, as just, ```as_json```. If that weren't the case, ```service_node_states_json``` would suffice.